.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: node_js
 
 node_js:
-  - '6'
-  - '7'
+  - '8.9'
```

Comment:
This is definitely not correct. We should test the Hapi v17 bits only on 8 and 9, but the rest also on Node 6.
We can drop 7.

Reply:
I agreed. But I'm not sure on how to configure Travis for this. Any hint?

Reply:
run the test everywhere, but then the projects that are 8+ should not run their tests if node < 8.